Expensify/App
# Refocus search input every time a person is selected/unselected

@MonilBhavsar, please review when you get the chance!
CC: @marcaaron, I wasn't sure whether it was fine to reference the ref of a child component like I do here. Wondering if you had ideas about a cleaner approach.

### Details
In the Group Chat creator, selecting a user to add will refocus the search bar. This will make it so you don't have to click the search bar again before you search for the next user.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/2452

### Tests
Followed the QA steps  on a local environment.

### QA Steps
1. Log into E.Cash and try to create a new Group by clicking on the bottom left.
2. When the side panel opens up, select one of the people or search for someone to add to the group chat.
3. Verify that each time you add a person by clicking/tapping on them, the serach bar gets refocused and you can search for the next person.


### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/19364431/116741643-60580f00-a9c4-11eb-9abe-8a46b40bbbc7.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/19364431/116740465-10c51380-a9c3-11eb-8d80-96298d1a7ac2.png)

#### Desktop
![image](https://user-images.githubusercontent.com/19364431/116741486-356dbb00-a9c4-11eb-95d4-ef600e63347f.png)

#### iOS
![image](https://user-images.githubusercontent.com/19364431/117046791-1aaa8780-acdf-11eb-965c-61c7fe48ee41.png)

#### Android
![image](https://user-images.githubusercontent.com/19364431/117086150-c53f9c00-ad19-11eb-8921-3140f3459969.png)
